### ISSUE TITLE
Crash when selecting copper atmo suit in exosuit forge menu when dupe assigned pyjammas.

### ISSUE THREAD

**Issue by Shakis87** [aki-art/ONI-Mods#29]
If you assign some pyjammas a a dupe (they don't need to actually wearing them, just assigned) then open the exosuit forge and try to select something to build (in my case I selected copper atmo suit. The game will crash.

The same may also be true for other "clothes" making buidings, I didn't check. 

Actually, I just went to get more reproducable data and log info for you but when I reactivated the mod and opened the exosuit forge it didn't crash. It may only be an issue when a dupe is assigned pyjammas, now that they're wearing them there's no crash, odd. I'll leave this here anyway and update if I have any more info/logs to present. 

Update, must have been a fluke it didn't crash that time.

![image](https://user-images.githubusercontent.com/33058163/209464479-c32bcbb6-504a-44ee-8e8c-8cd1b246b6e4.png)

Tail end Player.log
```
Unloading 6 unused Assets to reduce memory usage. Loaded Objects now: 817188.
Total: 2141.172400 ms (FindLiveObjects: 52.274000 ms CreateObjectMapping: 19.190400 ms MarkObjects: 2068.949700 ms  DeleteObjects: 0.758100 ms)

[10:04:26.899] [1] [INFO] DeconstructToolDefault is BUILDINGS!
[10:04:49.317] [1] [WARNING] Can't get sprite for type BalloonStand (UnityEngine.GameObject) (no KBatchedAnimController)
[10:04:49.694] [1] [WARNING] Can't get sprite for type BalloonStand (UnityEngine.GameObject) (no KBatchedAnimController)
[10:04:50.090] [1] [WARNING] Can't get sprite for type BalloonStand (UnityEngine.GameObject) (no KBatchedAnimController)
[10:04:50.418] [1] [WARNING] True Timelapse.OnNewDay but worldsToScreenshot is not empty
[10:04:55.254] [131] [INFO] [PLib/FastSave] Background save to: C:\Users\tjp_2\Documents\Klei\OxygenNotIncluded\cloud_save_files\76561197983833126\pee on me\auto_save\pee on me Cycle 585.sav
[10:04:55.259] [131] [INFO] [PLib/FastSave] Deleting old autosave: C:\Users\tjp_2\Documents\Klei\OxygenNotIncluded\cloud_save_files\76561197983833126\pee on me\auto_save\pee on me Cycle 575.sav
[10:04:55.885] [131] [INFO] [PLib/FastSave] Background save complete
[10:04:56.056] [133] [INFO] [PLib/FastSave] Saving screenshot to C:\Users\tjp_2\Documents\Klei\OxygenNotIncluded\cloud_save_files\76561197983833126\pee on me\auto_save\pee on me Cycle 585.png
[10:14:31.274] [1] [WARNING] True Timelapse.OnNewDay but worldsToScreenshot is not empty
[10:14:35.890] [165] [INFO] [PLib/FastSave] Background save to: C:\Users\tjp_2\Documents\Klei\OxygenNotIncluded\cloud_save_files\76561197983833126\pee on me\auto_save\pee on me Cycle 586.sav
[10:14:35.893] [165] [INFO] [PLib/FastSave] Deleting old autosave: C:\Users\tjp_2\Documents\Klei\OxygenNotIncluded\cloud_save_files\76561197983833126\pee on me\auto_save\pee on me Cycle 576.sav
[10:14:36.481] [166] [INFO] [PLib/FastSave] Saving screenshot to C:\Users\tjp_2\Documents\Klei\OxygenNotIncluded\cloud_save_files\76561197983833126\pee on me\auto_save\pee on me Cycle 586.png
[10:14:36.524] [165] [INFO] [PLib/FastSave] Background save complete
[10:28:00.478] [1] [WARNING] Failed to Unsubscribe event handler: StationaryChoreRangeVisualizer+<>c.Void <.cctor>b__29_0(StationaryChoreRangeVisualizer, System.Object)
Not subscribed to event
[10:28:00.478] [1] [WARNING] Failed to Unsubscribe event handler: StationaryChoreRangeVisualizer+<>c.Void <.cctor>b__29_1(StationaryChoreRangeVisualizer, System.Object)
Not subscribed to event -1643076535
[10:28:00.478] [1] [WARNING] Failed to Unsubscribe event handler: StationaryChoreRangeVisualizer+<>c.Void <.cctor>b__29_0(StationaryChoreRangeVisualizer, System.Object)
Not subscribed to event
[10:28:00.478] [1] [WARNING] Failed to Unsubscribe event handler: StationaryChoreRangeVisualizer+<>c.Void <.cctor>b__29_1(StationaryChoreRangeVisualizer, System.Object)
Not subscribed to event -1643076535
[10:28:13.174] [1] [WARNING] True Timelapse.OnNewDay but worldsToScreenshot is not empty
[10:28:17.489] [218] [INFO] [PLib/FastSave] Background save to: C:\Users\tjp_2\Documents\Klei\OxygenNotIncluded\cloud_save_files\76561197983833126\pee on me\auto_save\pee on me Cycle 587.sav
[10:28:17.492] [218] [INFO] [PLib/FastSave] Deleting old autosave: C:\Users\tjp_2\Documents\Klei\OxygenNotIncluded\cloud_save_files\76561197983833126\pee on me\auto_save\pee on me Cycle 577.sav
[10:28:18.134] [218] [INFO] [PLib/FastSave] Background save complete
[10:28:18.138] [219] [INFO] [PLib/FastSave] Saving screenshot to C:\Users\tjp_2\Documents\Klei\OxygenNotIncluded\cloud_save_files\76561197983833126\pee on me\auto_save\pee on me Cycle 587.png
[10:29:25.963] [1] [INFO] Saved to [C:\Users\tjp_2\Documents\Klei\OxygenNotIncluded\cloud_save_files\76561197983833126\pee on me\pee on me.sav]
[10:29:26.997] [225] [INFO] [PLib/FastSave] Saving screenshot to C:\Users\tjp_2\Documents\Klei\OxygenNotIncluded\cloud_save_files\76561197983833126\pee on me\pee on me.png
[10:29:30.457] [1] [ERROR] Could not find Personality: 0x0
Could not find Personality: 0x0
NullReferenceException: Object reference not set to an instance of an object
  at MinionStartingStats.CreateBodyData (Personality p) [0x0000f] in <d13cbb0b55a94ef09bb55c2436a6b8ee>:0 
  at (wrapper dynamic-method) Accessorizer.Accessorizer.ApplyMinionPersonality_Patch0(Accessorizer,Personality)
  at FullBodyUIMinionWidget.SetDefaultPortraitAnimator () [0x0004c] in <d13cbb0b55a94ef09bb55c2436a6b8ee>:0 
  at SelectedRecipeQueueScreen.RefreshMinionDisplayAnim () [0x00047] in <d13cbb0b55a94ef09bb55c2436a6b8ee>:0 
  at SelectedRecipeQueueScreen.ToggleAndRefreshMinionDisplay () [0x0000b] in <d13cbb0b55a94ef09bb55c2436a6b8ee>:0 
  at SelectedRecipeQueueScreen.SetRecipe (ComplexFabricatorSideScreen owner, ComplexFabricator target, ComplexRecipe recipe) [0x000cf] in <d13cbb0b55a94ef09bb55c2436a6b8ee>:0 
  at ComplexFabricatorSideScreen.ToggleClicked (KToggle toggle) [0x000c7] in <d13cbb0b55a94ef09bb55c2436a6b8ee>:0 
  at ComplexFabricatorSideScreen+<>c__DisplayClass36_1.<Initialize>b__1 () [0x00000] in <d13cbb0b55a94ef09bb55c2436a6b8ee>:0 
  at Util.Signal (System.Action action) [0x00003] in <fa50cdddb14a483d94f547557a674a42>:0 
  at KToggle.Click () [0x000a4] in <fa50cdddb14a483d94f547557a674a42>:0 
  at KToggle.OnPointerClick (UnityEngine.EventSystems.PointerEventData eventData) [0x0002c] in <fa50cdddb14a483d94f547557a674a42>:0 
  at UnityEngine.EventSystems.ExecuteEvents.Execute (UnityEngine.EventSystems.IPointerClickHandler handler, UnityEngine.EventSystems.BaseEventData eventData) [0x00007] in <0b23117e1c7442f8b1842fe2f3bf6fcb>:0 
  at UnityEngine.EventSystems.ExecuteEvents.Execute[T] (UnityEngine.GameObject target, UnityEngine.EventSystems.BaseEventData eventData, UnityEngine.EventSystems.ExecuteEvents+EventFunction`1[T1] functor) [0x0006c] in <0b23117e1c7442f8b1842fe2f3bf6fcb>:0 
UnityEngine.DebugLogHandler:Internal_LogException(Exception, Object)
UnityEngine.DebugLogHandler:LogException(Exception, Object)
UnityEngine.Logger:LogException(Exception, Object)
UnityEngine.Debug:LogException(Exception)
UnityEngine.EventSystems.ExecuteEvents:Execute(GameObject, BaseEventData, EventFunction`1)
UnityEngine.EventSystems.StandaloneInputModule:ReleaseMouse(PointerEventData, GameObject)
UnityEngine.EventSystems.StandaloneInputModule:ProcessMousePress(MouseButtonEventData)
UnityEngine.EventSystems.StandaloneInputModule:ProcessMouseEvent(Int32)
UnityEngine.EventSystems.StandaloneInputModule:ProcessMouseEvent()
UnityEngine.EventSystems.StandaloneInputModule:Process()
UnityEngine.EventSystems.EventSystem:Update()

[10:29:30.626] [1] [INFO] Checking which steam branch we're on. Got: []
```

**Comment by meelock** [comment]
It seems not to crash for the first cycle or so, then starts crashing when you click on any exosuit in the forge after that. Very weird.

**Comment by duneish** [comment]
For me the Game crashes after I open the Skills menu and then click on a recipe in the Exosuit Forge. I don't have any pajamas in my colony.

**Comment by aki-art** [comment]
Fixed in v1.2.9.2, live on workshop. Please use [Mod Updater](https://steamcommunity.com/sharedfiles/filedetails/?id=2018291283) to make sure it's the correct version.
